spacy/pipeline/edit_tree_lemmatizer.py

```diff
@@ -377,3 +394,7 @@ def _pair2label(self, form, lemma, add_label=False):
             self.tree2label[tree_id] = len(self.cfg["labels"])
             self.cfg["labels"].append(tree_id)
         return self.tree2label[tree_id]
+
+    @property
+    def activations(self):
+        return ["probs", "guesses"]
```

Comment:
We didn't and we should :).

Reply:
I have to say that the `List[str]` option and the `predict`/`set_annotations` dict format does not really ... inspire confidence about the long-term stability of this setting.

Reply:
I have a similar gut feeling, but not sure what a better alternative is. We could have some sort of generic `Enum` storing potential values across all components, even though that will contain a few highly component-specific ones?

Reply:
Also, a 'sealed' enum makes it harder for community to provide activations for their own pipe. Maybe a per-pipe enum? All the tagger-based components could share the same enum.

Reply:
How bad would it be for this setting to just be `bool` instead so that we don't have to expose this?

Reply:
Hm, so basically an all-or-nothing and let the users iterate over the dict to get the keys instead? It's an option - like how often would people actually go in and adjust which activations would be stored...

Reply:
The `Union[bool, List[str]]` came out of an earlier discussion. One of the benefits of giving the user the option of what to store is that particular activations for some pipes may be large arrays and would only be garbage collected when the `Doc` is deallocated (or the user explicitly deletes the corresponding keys). This could be a waste if they need another activation which is much smaller.

I don't think the size is an issue for our current components yet. `probs` is larger than `guesses`, but I think when people want activations, they want the larger `probs` anyway (since they are already getting the classes).

I think we could make it a `bool` now. If it turns out to be too limiting, we could always make it `Union[bool, List[str]]` or `Union[bool, List[SomeEnumType]]` later.

Reply:
I think trying to make this too configurable is making it too complicated and hard to maintain. It's so much easier if this stays an internal implementation detail.

Reply:
Yea, I know we discussed this earlier but it does make things quite a bit more complex in the code + some worries about maintainability/stability. So it's good to consider the trade-off to how often users would actually need to customize in more detail than on/off. 

Fine by me to have it as a boolean for now - it would also mean that existing configs wouldn't break if we expand to `Union[bool, List[str]]` later.

Reply:
I still have to rename the property, but it is a `bool` now.